assets/src/hooks/useDetour.ts

```diff
@@ -139,7 +152,11 @@ export const useDetour = ({ routePatternId, shape }: OriginalRoute) => {
 
   return {
     /** The current state of the detour machine */
-    state,
+    state: snapshot.matches({ "Detour Drawing": "Editing" })
+      ? DetourState.Edit
+      : snapshot.matches({ "Detour Drawing": "Share Detour" })
+      ? DetourState.Finished
+      : DetourState.Edit,
```

Comment:
```suggestion
      : DetourState.Finished
```

Not a huge deal, but this confused me because it made me think that there was some other state where we'd want `DetourState` to be `Edit` (of course, this is really just an argument for getting rid of `state` entirely, since the state machine can handle that now)

Reply:
*nods* I hated this, I just went with something and moved on to other things, I think https://github.com/mbta/skate/pull/2615 addresses this? But I can apply this in the mean time

Reply:
Yeah if it were going to be long-lived code, then this would matter more, but since this is on its way out, I don't think it matters either way.

Reply:
I just cleaned this up in https://github.com/mbta/skate/pull/2616 => https://github.com/mbta/skate/compare/c5999e059f2a5db282837cb620a18c2168166476..ed078862225b57c93cc6a8ad7d4bf78fdd5d36b7